### DISCUSSION TITLE
I have an issue

### DISCUSSION THREAD

**Discussion by Saadthe4205R** [AammarTufail/python_ka_chilla2023#39] [Q&A]
i have written the command but it is not working

![Screenshot (161)](https://user-images.githubusercontent.com/122175344/211165906-d087a263-50da-41fd-aed4-0a442edabad3.png)

**Comment by SahilJatoi744** [comment]
Which command is not working, brother?

**Reply by Saadthe4205R** [reply]
last one
![Screenshot (162)](https://user-images.githubusercontent.com/122175344/212296190-0731eaaa-e852-49bc-9c7f-1d1b16ad5688.png)

**Comment by sabdul4** [comment]
df.describe()
same issue balke mere pas tou sirf weight aa rha h

**Reply by SahilJatoi744** [reply]
tou apke datasets ke columns ka datatype numeric nh hoga is waja se issue araha ho

**Reply by Hashirahmad996** [reply]
first check df.info() then if height and wieght columns data type object then convert into int or float.

**Reply by sabdul4** [reply]
How to convert??

**Reply by Saadthe4205R** [reply]
ok but jesa Sir Amar Tufail ka aa rha tha wesa nahi aa rha or us me favorite dish ka column bhi tha

this is mine👇
![](https://user-images.githubusercontent.com/122175344/211165906-d087a263-50da-41fd-aed4-0a442edabad3.png)
of sir 👇
![Screenshot (164)](https://user-images.githubusercontent.com/122175344/212297490-84060ef6-42f1-4ed5-b75a-e250be722aca.png)
